app/src/pages/Organization.tsx

```diff
@@ -14,6 +14,7 @@ import {
 import { useTranslation } from "react-i18next";
 import { Link } from "../components/Link";
 import { DOCUMENT_TYPES, DOCUMENTS } from "./documents";
+import { LINKS } from "./links";
 
 const DIRECTORS = [
```

Comment:
You are talking about the `DIRECTORS` array right ? If yes I agree with you but I also think it's a bit out of scope. But since the PR is already so big I guess it doesn't really matter anymore haha